CHANGELOG.md

```diff
@@ -6,19 +6,52 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
 
 ___Note:__ Yet to be released changes appear here._
 
-## General
+## 5.20.0
 
+### General
+
+* `FEAT`: allow users to re-open recently used files ([#3917](https://github.com/camunda/camunda-modeler/issues/3917))
+* `FEAT`: confirm on unsaved files before reload ([#3886](https://github.com/camunda/camunda-modeler/issues/3886), [#3240](https://github.com/camunda/camunda-modeler/issues/3240))
+* `FEAT`: rework report feedback widget ([#4086](https://github.com/camunda/camunda-modeler/issues/4086))
+* `FEAT`: improve resizer and popup editor toggle interaction ([#3895](https://github.com/camunda/camunda-modeler/issues/3895))
+* `FIX`: correct user forum link ([#4085](https://github.com/camunda/camunda-modeler/issues/4085))
+* `CHORE`: track popup editor usage ([#4004](https://github.com/camunda/camunda-modeler/issues/4004))
 * `DEPS`: update to `@bpmn-io/form-js@1.7.0`
 * `DEPS`: update to `@camunda/form-linting@0.15.0`
 * `DEPS`: update to `@camunda/form-playground@0.13.0`
+* `DEPS`: update to `bpmn-js@16.4.0`
```

Comment:
Some dependencies seem outdated. For example, bpmn-js latest is [v16.5.0](https://www.npmjs.com/package/bpmn-js?activeTab=versions). Are we okay with that?

Reply:
Yes we are.

Reply:
[That one](https://github.com/bpmn-io/bpmn-js/blob/develop/CHANGELOG.md#1650) arrived after the code freeze, and hence did not make it.

Reply:
I'm not aware of other dependencies that are outdated.

Reply:
👍🏻